daml-foundations/daml-ghc/daml-stdlib-src/DA/Math.daml

```diff
@@ -0,0 +1,183 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+daml 1.2
+-- | Math - Utility Math functions for `Decimal`
+-- | The this library is designed to give good precision, typically giving 9 correct decimal places.
+-- | The numerical algorithms used are efficient, but still run with many iterations to achieve
```

Comment:
Only put `|` on the first line, the others don't need it (and having it is harmful).